src/sentry/api/endpoints/organization_events.py

```diff
@@ -55,7 +55,7 @@ def get(self, request: Request, organization) -> Response:
         performance_use_metrics = features.has(
             "organizations:performance-use-metrics", organization=organization, actor=request.user
         )
-        metrics_enhanced = referrer in METRICS_ENHANCED_REFERRERS and performance_use_metrics
+        metrics_enhanced = request.GET.get("metricsEnhanced") == "1" and performance_use_metrics
         referrer = (
```

Comment:
@wmak Should `referrer` be set to one of the values in `METRICS_ENHANCED_REFERRERS` in cases when `metrics_enhanced` is `True`? It could be helpful for debugging. 

Looks like `referrer` will fallback to `"api.organization-events-v2"`.

Reply:
Hmm it should be okay, the metric queries already get a suffix. I'd prefer to keep the same fallback referrer so we know that we've fallen back and there's potentially somewhere on the frontend we need to introduce a new referrer

Reply:
We should just append `.metrics` to the referrer here or in the frontend imo

Reply:
Ye :+1: https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/metrics_enhanced_performance.py#L69